src/dapp/libexec/dapp/dapp---use

```diff
@@ -31,12 +38,14 @@ shift
 
 [[ "$#" -gt 0 ]] || usage
 
-if [ -z "$override" ]; then
-
-dapp "$@"
-
+if [[ -z "$bin" ]]; then
+  echo >&2 "${0##*/}: Could not find ${solc} in your path."
+  echo >&2 "${0##*/}: Try \`nix-env -f . -iA solc-static-versions.$solc\`"
```

Comment:
A local dapptools checkout is no longer the default workflow. This should probably be something like:

```suggestion
  echo >&2 "${0##*/}: Try \`nix-env -f https://github.com/dapphub/dapptools/archive/master.tar.gz -iA solc-static-versions.$solc\`"
```

Reply:
Could we also use the `DAPPTOOLS` that is defined in the nix wrapper?

```
echo >&2 "${0##*/}: Try \`nix-env -f $DAPPTOOLS -iA solc-static-versions.$solc\`"
```